nmrfx-analyst/src/main/java/org/nmrfx/analyst/peaks/PeakFitting.java

```diff
@@ -158,19 +163,22 @@ public double jfitLinkedPeak(Peak peak, String fitMode) {
             if ((bounds == null) && (multiplet != null)) {
                 bounds = Multiplets.getBoundsOfPeakDims(multiplet.getAbsComponentList(), 1.5, regionShift);
             }
-            int[] dims = {0};
-            double[][] limits = new double[1][2];
-            limits[0][0] = bounds[0];
-            limits[0][1] = bounds[1];
-            List<Peak> peaks = peak.peakList.locatePeaks(limits, dims);
-            List<PeakDim> peakDims = new ArrayList<>();
-            for (Peak peakA : peaks) {
-                peakDims.add(peakA.getPeakDim(0));
+            if (bounds != null) {
+                int[] dims = {0};
+                double[][] limits = new double[1][2];
+                limits[0][0] = bounds[0];
+                limits[0][1] = bounds[1];
+                List<Peak> peaks = peak.peakList.locatePeaks(limits, dims);
+                List<PeakDim> peakDims = new ArrayList<>();
+                for (Peak peakA : peaks) {
+                    peakDims.add(peakA.getPeakDim(0));
+                }
+                value = fitPeakDims(peakDims, mode, bounds, fitMode);
+            } else {
+                log.warn("Unable to get bounds to fit peaks.");
```

Comment:
Since the previous behaviour was to just print the exception, just added a check for bounds being null and if it was, logged a message